evals/cli/oaieval.py

```diff
@@ -26,10 +24,8 @@ def _purple(str):
 
 def get_parser() -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(description="Run evals through the API")
-    parser.add_argument("model", type=str, help="Name of a completion model.")
+    parser.add_argument("completion_fn", type=str, help="One or more CompletionFn URLs, separated by commas (,). The format of a CompletionFn URL can be two forms: 1) an OpenAI API model followed by query parameters (e.g. `gpt-3.5-turbo?api_key=..`) or 2) a path to a Python class followed by query parameters (e.g. `evals.api:OpenAICompletionFn?model=text-davinci-003`).")
```

Comment:
Looking at the error, realize it's independent of the multiple completion_fns but bc `OpenAICompletionFn` does not like the chat prompt. I think we should change this assert to be the same as the one for `OpenAIChatCompletionFn` because `to_openai_create_prompt` should be able to handle converting the Chat format prompt into strings

Reply:
Changed to be same assert as OpenAIChatCompletionFn